### PR TITLE
[Feature] Add file

### PR DESCRIPTION
**Pull request by glukhove**

### CODE REVIEW

File: ernest_file [hunk 1]
```diff
@@ -0,0 +1 @@
+ernest
```
Comment (glukhove) [inline ernest_file:1]: Nice job!

**Review by glukhove — COMMENTED** [review]
Very nice job!

File: ernest_file (hunk 1 above)
Comment (AlexeyGankov) [reply ernest_file:1]: OK. approved

**Review by AlexeyGankov — COMMENTED** [review]
OK

**Pull request merged**